fe/fe-core/src/main/java/com/starrocks/sql/analyzer/SelectAnalyzer.java

```diff
@@ -230,6 +231,34 @@ private List<Expr> analyzeSelect(SelectList selectList, Relation fromRelation, A
                             INTERNAL_ERROR);
                 }
 
+                List<String> excludedColumns = item.getExcludedColumns();
```

Comment:
can we use this syntax in any contexts ? like CTE, subquery, select files ? 
if not, we need to declare these limitations in the document. 

```sql
-- producer of CTE 
with cte1 as ( select t1.* EXCLUDE (c1) from t1) ....

-- consumer of CTE
with cte1 as (select * from t1) select * EXCLUDE (c1) from cte1;

-- subquery 
select * from (select * EXCLUDE(c1) from t1)r;

- FILES: columns are automatically derived based on the data fields present within the file itself.
- select * EXCLUDE (id) from FILES(path='s3://xxxxx'); 
```

Reply:
I have tested the first two syntaxes and both work.
But thirdly, I don't have an S3 test environment.